test/integration_test/snapshot_restore_test.go

```diff
@@ -79,6 +79,7 @@ func cloudSnapshotRestoreTest(t *testing.T) {
 		scheduler.ScheduleOptions{AppKeys: []string{"mysql-cloudsnap-restore"}})
 	require.NoError(t, err, "Error scheduling task")
 	require.Equal(t, 1, len(snapCtx), "Only one task should have started")
+	verifyGroupSnapshot(t, snapCtx[0], defaultWaitTimeout)
```

Comment:
This is not a group snapshot

Reply:
The API name needs to be changed here, it just inspects the volumes from the context without differentiating between group or single volume snapshots.

Reply:
Changed method to validate snapshot in this case.